suite-common/connect-init/src/actions/prepareConnectInit.ts

```diff
@@ -0,0 +1,70 @@
+import {
+    ActionCreatorWithoutPayload,
+    ActionCreatorWithPayload,
+    AnyAction,
+    createAsyncThunk,
+} from '@reduxjs/toolkit';
+
+import TrezorConnect, { ConnectSettings, Manifest } from '@trezor/connect';
+
+import { cardanoConnectPatch } from '../cardanoConnectPatch';
+import { lockDeviceConnectPatch } from '../lockDeviceConnectPatch';
+import { bindConnectActionsListeners, ConnectEventAction } from '../bindConnectActionsListeners';
+
+type Selector<TReturnValue> = (state: any) => TReturnValue;
+type SuiteCompatibleAction<TPayload = void> = (
+    payload: TPayload,
+) => AnyAction | ActionCreatorWithPayload<TPayload> | ActionCreatorWithoutPayload;
+
+export const CONNECT_INIT_ACTION_TYPE = '@suite-common/connect-init/init';
+
+export const prepareConnectInit = createAsyncThunk<
```

Comment:
We should start suffixing all thunk with thunk. `prepareConnectInit` => `prepareConnectInitThunk`

Reply:
https://github.com/trezor/trezor-suite/pull/5760/commits/979f44f2e30665c851548bc75ae31ae9d9bbba9b